chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDescription.java

```diff
@@ -0,0 +1,103 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+/**
+ * Describes {@link JTest} prototype.
+ */
+public class JTestDescription {
+
+    private String id;
+    private String comment;
+    private Iterable endpoints;
+    private Iterable queries;
+
+    private JTestDescription(Builder builder) {
+        this.id = builder.id;
+        this.comment = builder.comment;
+        this.endpoints = builder.endpoints;
+        this.queries = builder.queries;
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private String id;
+        private String comment;
+        private Iterable endpoints;
+        private Iterable queries;
+
+        private Builder() {
+
+        }
+
+        /**
+         * Sets id for the test prototype.
+         *
+         * @param id the id of the test prototype.
+         */
+        public Builder withId(String id) {
+            this.id = id;
+            return this;
+        }
+
+
+        /**
+         * Sets human readable comment for the test prototype.
+         *
+         * @param comment the comment of the test prototype.
+         */
+        public Builder withComment(String comment) {
+            this.comment = comment;
+            return this;
+        }
+
+        /**
+         * Sets end points for the tests inherited from the current test prototype.
+         *
```

Comment:
Sets end points (where load will be applied during performance test) for the tests using this test prototype